api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -201,18 +202,22 @@ public synchronized Order saveOrder(Order order, OrderContext orderContext) thro
 		}
 		
 		if (DISCONTINUE != order.getAction()) {
-			List<Order> activeOrders = getActiveOrders(order.getPatient(), null, order.getCareSetting(), null);
-			for (Order activeOrder : activeOrders) {
-				//Reject if there is an active drug order for the same orderable with overlapping schedule
-				if (areDrugOrdersOfSameOrderableAndOverlappingSchedule(order, activeOrder)) {
-					throw new APIException("Order.cannot.have.more.than.one", (Object[]) null);
-				}
+ 			List<Order> activeOrders = getActiveOrders(order.getPatient(), null, order.getCareSetting(), null);
+			List<String> parallelOrders = Collections.emptyList();
+			if(orderContext!=null && orderContext.getAttribute(PARALLEL_ORDERS)!=null){
+				parallelOrders= Arrays.asList((String[])orderContext.getAttribute(PARALLEL_ORDERS));
 			}
-		}
+ 			for (Order activeOrder : activeOrders) {
+ 				//Reject if there is an active drug order for the same orderable with overlapping schedule
+				if (!parallelOrders.contains(activeOrder.getUuid()) && areDrugOrdersOfSameOrderableAndOverlappingSchedule(order, activeOrder)) {
+					throw new AmbiguousOrderException("Order.cannot.have.more.than.one", (Object[]) null);
```

Comment:
It is pointless to pass in a null value here when there is a constructor that takes no argument, i will remove it